doc/lsst.source.injection/reference/11_make_injection_pipeline.rst

```diff
@@ -22,6 +22,11 @@ Typically, a pipeline reference file will be a pipeline that is being used to re
 Either the :doc:`make_injection_pipeline <../scripts/make_injection_pipeline>` command line script or the associated :py:func:`~lsst.source.injection.make_injection_pipeline` Python function may be used to generate a fully qualified injection pipeline.
 Examples on this page illustrate the use of both methods.
 
+.. note::
+  Two dynamic source injection pipelines are automatically generated inside the ``drp_pipe`` repository.
```

Comment:
Can you use 4-space tabs for indenting the block here? That would be in-keeping with the rest of the docs. 

I think you also need a new-line between `.. note::` and the text.